test/built-ins/Object/create/15.2.3.5-0-2.js

```diff
@@ -4,12 +4,6 @@
 /*---
 es5id: 15.2.3.5-0-2
 description: Object.create must exist as a function taking 2 parameters
-includes: [runTestCase.js]
 ---*/
 
-function testcase() {
-  if (Object.create.length === 2) {
-    return true;
-  }
- }
-runTestCase(testcase);
+assert.sameValue(Object.create.length, 2, 'Object.create.length: ');
```

Comment:
We don't use any custom reporter, but I may see some future within a website running the tests and reporting each assertion. These messages wouldn't make much change. 

In other perspective: I like (and I'm used to) assertion messages explaining what they do. Here, these messages are only functional on failure cases for the current reporter. And in this case, even the space in the end could be trimmed.